chap-cheri-x86-64.tex

```diff
@@ -134,6 +133,10 @@ \subsection{Unique Architectural Features}
  \item Requests for most non-monotonic capability transformations
    clear the tag on the destination capability rather than raising an
    exception.
+ \item \RIP{} is the full integer value (virtual address) of \CIP{}
```

Comment:
After #34 lands these will no longer be unique features.

Reply:
I will leave them in for now though since they are still unique until RISC-V is updated to handle offsetting differently.